node/consensus/data/consensus_frames.go

```diff
@@ -23,29 +23,25 @@ func (e *DataClockConsensusEngine) collect(
 ) (*protobufs.ClockFrame, error) {
 	e.logger.Info("collecting vdf proofs")
 
+	e.syncingStatus = SyncStatusSynchronizing
```

Comment:
This was what I was referring to – the sync status shouldn't be set unless sync is being invoked, due to the possibility the node could be an active frame prover (obviously not at the moment but after pre2.0 proofs are done and we can unshackle ourselves from the beacon model) – synchronizing state will halt processing of other messages. I think the approach is awesome, this is the only blocking item.

Reply:
Should be fixed now.